mmv1/templates/terraform/constants/workbench_instance.go

```diff
@@ -195,3 +195,48 @@ func waitForWorkbenchOperation(config *transport_tpg.Config, d *schema.ResourceD
 	return nil
 }
 <% end -%>
+
+func resizeWorkbenchInstanceDisk(config *transport_tpg.Config, d *schema.ResourceData, project string, userAgent string, isBoot bool) (error) {
```

Comment:
Do you mind putting the whole function inside `<% unless compiler == "terraformgoogleconversion-codegen" -%>` ?